RHIDP-984: Fixed GitHub login failure (#64)
redhat-developer/red-hat-developers-documentation-rhdh: installation/proc-install-rhdh-helm.adoc

. Click *Create*.

> . Copy the OpenShift router host (for example: `apps.<clusterName>.com`) to *Root Schema* -> *global* -> *Shorthand for users who do not want to specify a custom HOSTNAME. Used ONLY with the DEFAULT upstream.backstage.appConfig value and with OCP Route enabled.*, and adjust the other values if needed.

Reviewer: This also needs to be updated.

```suggestion
. Copy the OpenShift router host (for example: `apps.<clusterName>.com`) to *Root Schema* -> *global* -> *Custom hostname shorthand, overrides `global.clusterRouterBase`, `upstream.ingress.host`, `route.host`, and url values in `upstream.backstage.appConfig`*, and adjust the other values if needed.
```

Author: Done
